kubernetes/cmsweb/services/dqmsquare.yaml

```diff
@@ -0,0 +1,196 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: dqm-square-k8
+  namespace: dqm
+spec:
+  selector:
+    app: dqm-square-mirror-server
+  ports:
+    - port: 8084
+      targetPort: 8084
+      name: dqm-square-mirror-server
+    - port: 18084
+      targetPort: 18084
+      name: dqm-square-mirror-server-mon
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata: 
+  name: dqm-square-mirror-server
+  namespace: dqm
+  labels:
+    app: dqm-square-mirror-server
+spec:
+  selector:
+    matchLabels:
+      app: dqm-square-mirror-server
+  replicas: 1
+  template: # This is the pod template
+    metadata:
+      labels:
+        app: dqm-square-mirror-server
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+      - name: dqm-square-mirror-server
+        image: registry.cern.ch/cmsweb/dqmsquare_mirror:latest #imagetag
```

Comment:
Please remove `:latest` from image

Reply:
done as well as for following occurrences